parcels/particlefile.py

```diff
@@ -0,0 +1,98 @@
+import numpy as np
```

Comment:
Would it make sense to rename this file to fileIO.py or something, and also add the reading and writing of netcdf files from field.py here?